core/src/test/scala/integration/kafka/api/TransactionsTest.scala

```diff
@@ -759,8 +762,107 @@ class TransactionsTest extends IntegrationTestHarness {
   }
 
   @ParameterizedTest
-  @ValueSource(strings = Array("zk", "kraft"))
-  def testFailureToFenceEpoch(quorum: String): Unit = {
+  @CsvSource(Array("kraft, true"))
```

Comment:
We can keep the original value source since isTV2Enabed is true.

Reply:
I could remove it for the TV2 enabled case since the default is set to true (but if anyone decides to change it at some point then this test would fail), but we would definitely have to specify false in the other test

Reply:
Hmm. I see. I wish there was a way we could pass the value into the test automatically. But I suppose not.